src/modules/endpoint/FeedbackEndpoint.js

```diff
@@ -0,0 +1,82 @@
+// @flow
```

Comment:
Did you remove the FeedbackEndpointBuilder? In the last PR I did not get that there are more than one endpoints. Can not find the comment but there was no discussion.

Reply:
Yes I removed it. There is just this FeedbackEndpoint.